birthday/README.md

```diff
@@ -1,92 +1,90 @@
-Birthday CC V2.0
-================
+# Birthdays
+This custom command adds a birthday functionality to your server, wishing members all the best on their respective date.
 
-# Info
+## Features
+* Send a congratulation message in the chat
+* Kick or Ban users younger than 13 years
+* Allow users to set their birthday date
```

Comment:
"Birthday date" sounds... repetitive to me. Birthday implies date to me, so I think it could just be "[...] to set their birthday". Your choice though, not a big deal.

Reply:
Agreed, sort of redundant. Will change it